artifactory/spec/specfiles.go

```diff
@@ -129,9 +128,6 @@ func ValidateSpec(files []File, isTargetMandatory, isSearchBasedSpec bool) error
 		if isTargetMandatory && !isTarget {
 			return errors.New("Spec must include target.")
 		}
-		if !isSearchBasedSpec && !isAql && !isPattern {
-			return errors.New("Spec must include either aql or pattern.")
```

Comment:
Please try to run upload with spec without pattern:
```
{
  "files": [
    {
      "target": "libs-release-local/"
    }
  ]
}
```

Reply:
Done.